gap/almostsimple.gi

```diff
@@ -649,6 +653,8 @@ RECOG.MakePSL2Hint := function( name, G )
               orblenlimit := 3*(name[3]+1) );
 end;
 
+# is used in FindHomMethodsProjective.ComputeSimpleSocle
+# it computes randomly (so maybe not) the non-abelian simple socle
```

Comment:
"so maybe not"? So maybe not randomly? Or do you mean to say "it could fail"?

I'd also spell it as "nonabelian"